### ISSUE TITLE
Double Category Issue

### ISSUE THREAD

**Issue by wheatprune98** [bluesabre/menulibre#189]
System info: `Debian 12 Bookworm MATE 1.26.0`
When you try to make a new launcher, it just shows **two category instead of one**. When you edit exist launcher, you will get **double category and double launcher**. Is this have problem with difference of placing `libremenu` .desktop files in `~/.local/share/applications/` rather than `/usr/share/applications` and because of this the system thinks the **.desktop files are duplicate instead of only-one launcher?**

**Comment by bluesabre** [comment]
Are you using the version of MenuLibre that currently ships with Debian (2.2.2) or the latest release (2.3.2)? There are some MATE-specific fixes that have been made since 2.2.2 that might resolve this issue.

As far as installing .desktop files in `~/.local/share/applications/` vs `/usr/share/applications`, this is the correct way of replacing a system .desktop file and MATE's menu-handling library should already support this.

**Comment by wheatprune98** [comment]
Yes. I am using the `menulibre 2.2.2` that ships with Debian. Maybe i need to get a new one via release and compile it for myself?

**Comment by billbeans** [comment]
Yeah, this happened to me on Debian 12, and it looks like MenuLibre creates a custom category in `~/.config/menus/applications-merged`. If you go inside and delete the custom file added by MenuLibre the duplicate category should get removed. Is there a way to make the program not do that? I don't get why all of these GUI desktop creators have so many problems. I already know how to make them manually, I just wanted something convenient and maybe nice to look at.

**Comment by dirkf** [comment]
Fixed in https://github.com/bluesabre/menulibre/commit/f882539a9fc420d1ba46056091c73e448e84eeb0 (2.3.0)?
